go/vt/mysqlctl/backup.go

```diff
@@ -156,7 +176,8 @@ func Backup(ctx context.Context, params BackupParams) error {
 	}
 
 	// The backup worked, so just return the finish error, if any.
-	backupDuration.Set(int64(time.Since(startTs).Seconds()))
+	stats.DeprecatedBackupDurationS.Set(int64(time.Since(startTs).Seconds()))
+	params.Stats.Scope(stats.Operation("backup")).TimedIncrement(time.Since(startTs))
```

Comment:
.scope return stats, do we not need to assigned it back ??

Reply:
Assign it back to `params`?

Reply:
yeah

Reply:
actually I re-read the code.. we are ok here..